tests/worksheets/src/test/scala/tests/worksheets/WorksheetSuite.scala

```diff
@@ -329,6 +329,23 @@ class WorksheetSuite extends BaseSuite {
        |""".stripMargin
   )
 
+  checkDiagnostics(
+    "dotty-interpolation".tag(OnlyScala3),
```

Comment:
rename test, one above is called the same

Reply:
Fixed!